app/_hub/kong-inc/jq/index.md

```diff
@@ -0,0 +1,101 @@
+---
+name: Kong jq
+publisher: Kong Inc.
+versions: 0.1.x
+beta: true
+
+desc: Transform JSON objects included in API requests or responses using jq programs.
+description: |
+  The Kong jq plugin enables arbitrary jq transformations on JSON objects included in API requests or responses.
+  The configuration accepts two sets of options: one for the request and another for the response.
+  For both the request and response, a jq program string can be included, along with some jq option flags
+  and a list of media types. One of the media types must included in the `Content-Type` header for
+  the jq program to run (the media type in the `Content-Type` header defaults to `application/json`).
+  In the response context, you also have the option to specify a list of status codes, one of which must match the response status code (the response status code defaults to `200`).
+
+  {:.note}
+  > **Note:** In the response context the entire body must be buffered to be processed. This requirement also
+  implies that the `Content-Length` header will be dropped if present, and the body transferred with chunked encoding.
+
+  See jq's documentation on [Basic filters](https://stedolan.github.io/jq/manual/#Basicfilters) for more information on writing programs with jq.
+
+enterprise: true
+type: plugin
+categories:
+ - transformations
+
+kong_version_compatibility:
+    enterprise_edition:
+      compatible: 
+        - 2.6.x
+
+params:
+  name: jq
+  service_id: true
+  route_id: true
+  consumer_id: true
+  yaml_examples: false
+  k8s_examples: false
+  konnect_examples: false
+  protocols: ["http","https"]
+  dbless_compatible:
+  config:
+    - name: request_jq_program
+      required: semi
+      datatype: string
+      description: |
+        The jq program to run on the request body. For example, `.[0] | { "X-Foo": .foo }`. 
+        Either `request_jq_program` or `response_jq_plugin` **must** be included in configuration.
```

Comment:
Should label either `request_jq_program` or `response_jq_plugin` as something to use in examples, then do the same for all of its properties.

For example, for this property you could set:
`value_in_examples: .[0] | { "X-Foo": .foo }`

 If you don't want examples _at all_ (ie if this plugin is too weird to show a generic generated config example for), you can disable examples entirely with `examples: false` instead of disabling each one individually.

Reply:
I will go ahead and disable examples here